src/vivarium_public_health/metrics/disability.py

```diff
@@ -116,3 +113,36 @@ def get_disability_weight_pipeline(self, builder: Builder) -> Pipeline:
 
     def disability_weight_aggregator(self, dw: pd.DataFrame) -> float:
         return (dw * to_years(self.step_size)).sum().squeeze()
+
+    ##################
+    # Report methods #
+    ##################
+
+    def report(
+        self,
+        measure: str,
+        results: pd.DataFrame,
+    ) -> None:
+        """Combine the measure-specific observer results and save to a single file."""
+        results_dir = Path(self.results_dir)
+        measure, cause = measure.split("due_to")
+        measure = measure.strip("_")
+        cause = cause.strip("_")
+        # Add extra cols
+        results["measure"] = measure
+        results["cause"] = cause
+        results["random_seed"] = self.random_seed
+        results["input_draw"] = self.input_draw
+        # Sort the columns such that the stratifications (index) are first
+        # and "value" is last and sort the rows by the stratifications.
+        other_cols = [c for c in results.columns if c != "value"]
```

Comment:
How do we know the columns we're looking for is called "value"? Where is this defined? Can there be a constant somewhere that is referenced?

Reply:
[It's defined in vivarium](https://github.com/ihmeuw/vivarium/blob/epic/MIC-4968-results-processing/src/vivarium/framework/results/context.py#L138). I'll make a constant